sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -52,7 +58,7 @@ public FormTrainingClient(Uri endpoint, AzureKeyCredential credential, FormRecog
         /// Trains a model from a collection of custom forms in a blob storage container.
         /// </summary>
         /// <param name="trainingFiles">An externally accessible Azure storage blob container Uri.</param>
-        /// <param name="useLabels"></param>
+        /// <param name="useLabels">If <c>true</c>, use label files to train a labeled model. Otherwise, a non-labeled model will be trained.</param>
```

Comment:
```suggestion
        /// <param name="useLabels">If <c>true</c>, use a label file created in the <link-to-label-tool-doc> to provide training-time labels for training a model. Otherwise, the model will be trained from forms only.</param>
```